palace/linalg/strumpack.cpp

```diff
@@ -8,25 +8,64 @@
 namespace palace
 {
 
+namespace
+{
+
+strumpack::CompressionType
+GetCompressionType(config::LinearSolverData::CompressionType type)
+{
+  switch (type)
+  {
+    case config::LinearSolverData::CompressionType::HSS:
+      return strumpack::CompressionType::HSS;
+    case config::LinearSolverData::CompressionType::BLR:
+      return strumpack::CompressionType::BLR;
+    case config::LinearSolverData::CompressionType::HODLR:
+      return strumpack::CompressionType::HODLR;
+    case config::LinearSolverData::CompressionType::ZFP:
+      return strumpack::CompressionType::LOSSY;
+    case config::LinearSolverData::CompressionType::BLR_HODLR:
+      return strumpack::CompressionType::BLR_HODLR;
+      break;
+    case config::LinearSolverData::CompressionType::ZFP_BLR_HODLR:
+      return strumpack::CompressionType::ZFP_BLR_HODLR;
+      break;
+    case config::LinearSolverData::CompressionType::NONE:
+    default:
```

Comment:
Generally for switching over enum `default:` should be avoided, as without it the compiler will warn in future if any unhandled enum cases are introduced.

Reply:
Good call, I'll address this.

Reply:
Resolved throughout in [d8bc2f1](https://github.com/awslabs/palace/pull/56/commits/d8bc2f1d1d5ed87110a5cb64f723ace3e3519388).